generators/typescript/sdk/generator/src/TemplateGenerator.ts

```diff
@@ -291,7 +295,9 @@ export class TemplateGenerator {
         }
         return FdrSnippetTemplate.Template.generic({
             imports: [],
-            templateString: this.getAsNamedParameterTemplate(name, `{\n${childTabs}${TEMPLATE_SENTINEL}\n${selfTabs}}`),
+            templateString: isUnionType
+                ? this.getAsNamedParameterTemplate(name, `${TEMPLATE_SENTINEL}\n`)
```

Comment:
Do you not need to persist the tabs here?
I think this might need to be something like  `this.getAsNamedParameterTemplate(name, ${childTabs}${TEMPLATE_SENTINEL}\n)`

Reply:
I don't think we do since the value will be on the same line rather than the next

Reply:
You should run some tests! I think you might have to since this template corresponds to multiple properties which will be on separate lines

Reply:
hey @armandobelardo -- it doesn't look like this breaks existing unions. since we need to move a little fast on this, planning to fix in a follow up and release an rc of the generator